packages/node/src/evm/contracts/airnodeRrp.test.ts

```diff
@@ -59,34 +59,34 @@ describe('AirnodeRrp', () => {
     expect.assertions(7);
 
     expect(Object.keys(airnodeRrpTopics).sort()).toEqual([
-      'ClientFullRequestCreated',
-      'ClientRequestCreated',
-      'ClientRequestFailed',
-      'ClientRequestFulfilled',
-      'WithdrawalFulfilled',
-      'WithdrawalRequested',
+      'MadeFullRequest',
+      'MadeTemplateRequest',
+      'FailedRequest',
+      'FulfilledRequest',
+      'FulfilledWithdrawal',
+      'RequestedWithdrawal',
     ]);
 
     // API calls
-    expect(airnodeRrpTopics.ClientRequestCreated).toEqual(
+    expect(airnodeRrpTopics.MadeTemplateRequest).toEqual(
       '0x8339fddbb81e588a9ed04dec82ee9ae6c7a185f44835adaaa2ace50ce3a14aaf'
```

Comment:
Haven't these values have changed too?

Reply:
Yes, probably but I was hoping to get the code in a state where I could run the tests and see what the new value would be. Do you think I should try to get this hash computed now and replace it?

Reply:
No need to if it's difficult to get. Can always fix this later